sdk/src/client/bindings/python/examples/08_data_block.py

```diff
@@ -3,12 +3,7 @@
 # Create an IotaClient instance
 client = IotaClient(nodes = ['https://api.testnet.shimmer.network'])
 
-options = {
-    # `hello` hex encoded
-    "tag": '0x68656c6c6f',
-    "data": '0x68656c6c6f',
-}
-
-# Create and post a block with a tagged data payload
-block = client.build_and_post_block(None, options)
+# Create and post a block with a tagged data payload. Data is 'hello' hex encoded
+block = client.build_and_post_block(
+    None, tag='0x68656c6c6f', data='0x68656c6c6f')
```

Comment:
What's that first `None` ? Shouldn't it be named ?

Reply:
It's the secret manager which is in position 0. So it will work. But you are right, it may be harder to understand. Should I make it named? Wdyt?

Reply:
Yes, named

Reply:
Done